src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/ECDiffieHellmanWrapper.cs

```diff
@@ -152,7 +152,7 @@ public override byte[] ExportEncryptedPkcs8PrivateKey(
 
         public override byte[] ExportSubjectPublicKeyInfo() => _wrapped.ExportSubjectPublicKeyInfo();
 
-        public override bool Equals(object? obj) => _wrapped.Equals(obj);
+        public override bool Equals(object? obj) => obj is ECDiffieHellmanWrapper wrapper && _wrapped.Equals(wrapper._wrapped);
```

Comment:
Alternatively, we could just remove the overrides entirely and let reference equality "just work" on the wrappers, instead of the wrapped.

Reply:
Yeah, probably better to just delete the overrides, and fix the test that complains we didn't override them.

Or just explicitly make it `ReferenceEquals(this, obj)`.

Reply:
As far as I can tell we don't have reflection tests for the wrappers. We only have them for the CryptoServiceProviders that are fake.

Might as well just get rid of it then.